pkg/agent/run.go

```diff
@@ -32,10 +33,10 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-const (
-	InternalIPLabel = "k3s.io/internal-ip"
-	ExternalIPLabel = "k3s.io/external-ip"
-	HostnameLabel   = "k3s.io/hostname"
+var (
+	InternalIPLabel = version.Program + ".io/internal-ip"
+	ExternalIPLabel = version.Program + ".io/external-ip"
+	HostnameLabel   = version.Program + ".io/hostname"
```

Comment:
This seems not very general purpose. Should we have a `version.ProgramDomain`? We can reverse as needed.